.github/workflows/ci.yml

```diff
@@ -25,3 +25,19 @@ jobs:
           bundler-cache: true
 
       - run: bundle exec rake
+
+  rubocop:
+    name: Run rubocop
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: rm Gemfile.lock
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.0'
```

Comment:
should match TargetRubyVersion

Reply:
rubocop doesn't care about host ruby, so I think it should be the version closes to the one on average developer machine to make `Gemfile.lock` relevant. Not sure about 3.0, but IMHO it should be no less than 2.6.

Reply:
hmmm true ... I usually keep it at the same version for symmetry, but should be fine 👍